src/System.Collections/tests/System.Collections.Tests.csproj

```diff
@@ -10,6 +10,7 @@
     <RootNamespace>System.Collections.Tests</RootNamespace>
     <DefineConstants Condition="'$(TargetGroup)'=='netstandard1.7'">$(DefineConstants);netstandard17</DefineConstants>
```

Comment:
You are going to need to add a build configuration in the System.Collections.Tests.builds file in order to get this netcoreapp1.1 build setup.